src/components/HomePage/MoviesSection/MoviesSection.vue

```diff
@@ -0,0 +1,135 @@
+<script>
+import { defineComponent } from 'vue';
+
+import MoviesSectionCard from '@/components/HomePage/MoviesSection/MoviesSectionCard.vue';
+
+const moviesList = [
+{
+id: 1,
+title: "Harry Potter and the Philosopher's Stone",
+genre: { id: 3, name: "Fantasy" },
+poster_url:
+    "https://images-na.ssl-images-amazon.com/images/I/713KEd-8jyL._AC_SL1500_.jpg",
+length: 159
+},
+{
+id: 2,
+title: "Star Wars Episode IV: A New Hope",
+genre: { id: 4, name: "Sci-Fi" },
+poster_url:
+    "https://img01.mgo-images.com/image/thumbnail/v2/content/MMVD85527F13D6BC8CB6518D4B8DF956FAD2.jpeg",
+length: 125
+},
+{
+id: 3,
+title: "You've got Mail",
+genre: { id: 5, name: "Romance" },
+poster_url: "https://miro.medium.com/max/520/0*JeA6QiQ6BzuNzKHP.jpg",
+length: 120
+}
+];
+
+export default defineComponent({
+    name: 'MoviesSection',
+    components: {
+        MoviesSectionCard,
+    },
+    data() {
+        return {
+            movies: moviesList,
+        }
+    },
+});
+</script>
+
+<template>
+    <section class="movies-section">
+        <div class="movies-section-header">
+            <h2 class="movies-section-header-text">soon<span class="movies-section-header-text--toggle"> in the cinema</span></h2>
+            <a href="#" class="movies-section-header-link">see all</a>
+        </div>
+        <div class="movies-section-content">
+            <MoviesSectionCard 
+                v-for="movie in movies" 
+                :key="movie.id" 
+                :movieTitle="movie.title" 
+                :movieImgUrl="movie.poster_url"
+                :movieLength="movie.length" 
+                :movieGenre="movie.genre.name"
+            />
+        </div>
+    </section>
+</template>
+
+<style lang="scss" scoped>
+.movies-section {
+    // display: flex;
+    // flex-direction: column;
+    // justify-content: center;
+    // align-items: center;
+    margin-bottom: 64px;
+
+    .movies-section-header {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-bottom: 24px;
+
+        .movies-section-header-text, .movies-section-header-link {
```

Comment:
Thanks for highlighting this problem of mine that I've been somewhat aware of - using "BEM-like" notation instead of the real deal BEM.. :).

Today we had a presentation regarding styling during which Oskar told us how to leverage BEM element & modifier name parts to keep the specificity down while still having the styles tightly targeted.

I intend to do a major style refactor throughout the whole project after I merge this PR - hope it's OK with You?

Reply:
Sure thing - plan your work. It's not a major issue unless it's really important, reusable component (this is obviously not one of that kind).

Reply:
Great, thanks!